test/compress/hoist_props.js

```diff
@@ -827,3 +827,25 @@ issue_2519: {
     }
     expect_stdout: "5.5"
 }
+
+toplevel_const: {
+    options = {
+        hoist_props: true,
+        reduce_vars: true,
+    }
+    input: {
+        const a = {
+            b: 1,
+            c: 2
+        };
+        console.log(a.b + a.c);
+    }
+    expect: {
+        const a = {
+            b: 1,
+            c: 2
+        };
+        console.log(a.b + a.c);
+    }
+    expect_stdout: "3"
+}
```

Comment:
Just for the heck of it, could you copy this test for a top level `let` and `var`?

Reply:
And another set of tests for `toplevel: true` for `var`/`let`/`const`?

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/2724/commits/0e6ac6d29348b4ab3f992c7661a55f7b3d59f4e4

Reply:
I guess the `toplevel: true` tests would be run by the reminify logic.

Reply:
> And another set of tests for toplevel: true for var/let/const?

There are existing tests in `hoist_props.js` that would convert those anyway.